contracts/clients/src/BlsSigner.ts

```diff
@@ -5,12 +5,25 @@ import BlsProvider from "./BlsProvider";
 import BlsWalletWrapper from "./BlsWalletWrapper";
 import { ActionData, bundleToDto } from "./signer";
 
+/**
+ * Dedicated class to manage the private key and BlsWalletWrapper instance.
+ */
+export class BlsSignerWrapper {
+  wallet: BlsWalletWrapper;
+  privateKey: string;
+
+  constructor(wallet: BlsWalletWrapper, privateKey: string) {
+    this.wallet = wallet;
+    this.privateKey = privateKey;
```

Comment:
Not actually using this property (`blsSignerWrapper.privateKey`) anywhere. `blsSignerWrapper.wallet.privateKey` is used instead (see `connectedUnchecked`). Wondering whether it is actually needed?